### DISCUSSION TITLE
Contributing Guide - Ultralytics YOLOv8 Docs

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#6265] [Docs]
# Contributing Guide - Ultralytics YOLOv8 Docs

Learn how to contribute to Ultralytics YOLO projects – guidelines for pull requests, reporting bugs, code conduct and CLA signing.

https://docs.ultralytics.com/help/contributing/

**Comment by jedi007** [comment]
I have read the CLA Document and I sign the CLA

**Reply by glenn-jocher** [reply]
Nice! I like the effort, but you actually don't need to write this here, this should be placed in a comment directly in the Pull Request (PR) that you open :)

**Comment by glenn-jocher** [comment]
Thanks for browsing our Contributing Guide! If you follow the principles set forth above then your code should be more readable and understandable by other users, and this close alignment with best practices will help us review your work and integrate it into our open-source repos more efficiently!
